testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2WD/sandy/config_forward.xml

```diff
@@ -0,0 +1,102 @@
+		<option name="config_time_varying_atmospheric_forcing_start_time">'2012-10-01_00:00:00'</option>
+		<option name="config_time_varying_atmospheric_forcing_reference_time">'2012-10-01_00:00:00'</option>
+		<option name="config_time_varying_atmospheric_forcing_cycle_start">'none'</option>
+		<option name="config_time_varying_atmospheric_forcing_interval">'01:00:00'</option>
+		<option name="config_time_varying_atmospheric_forcing_ramp">20.0</option>
+		<option name="config_AM_pointwiseStats_enable">.true.</option>
+		<option name="config_check_ssh_consistency">.false.</option>
+		<option name="config_use_wetting_drying">.true.</option>
+		<option name="config_prevent_drying">.true.</option>
+		<option name="config_drying_min_cell_height">0.1</option>
+		<option name="config_verify_not_dry">.true.</option>
+		<option name="config_zero_drying_velocity">.true.</option>
+		<option name="config_thickness_flux_type">'upwind'</option>
+		<option name="config_use_const_visc">.true.</option>
+		<option name="config_vert_visc">1.0e4</option>
+	</namelist>
+
+	<streams name="streams.ocean" keep="immutable" mode="forward">
+		<stream name="mesh">
+			<attribute name="filename_template">input.nc</attribute>
+		</stream>
+		<stream name="input">
+			<attribute name="filename_template">input.nc</attribute>
+		</stream>
+		<stream name="restart">
+			<attribute name="output_interval">0002_00:00:00</attribute>
+		</stream>
+		<stream name="pointLocationsInput">
+			<attribute name="filename_template">points.nc</attribute>
+			<attribute name="type">input</attribute>
+			<attribute name="input_interval">initial_only</attribute>
+			<attribute name="runtime_format">single_file</attribute>
+			<attribute name="mode">forward;analysis</attribute>
+                        <add_contents>
+				<member name="pointCellGlobalID" type="var"/>
+                        </add_contents>
+		</stream>
+		<template file="minimal_output.xml" path_base="script_core_dir" path="templates/streams"/>
+		<stream name="output">
+			<attribute name="output_interval">12:00:00</attribute>
+			<add_contents>
+				<member name="atmosphericPressure" type="var"/>
+				<member name="windSpeedMagnitude" type="var"/>
+				<member name="ssh" type="var"/>
+				<member name="kineticEnergyCell" type="var"/>
+			</add_contents>
+			<remove_contents>
+				<member name="tracers"/>
+                                <member name="normalVelocity"/>
+			</remove_contents>
+		</stream>
+		<stream name="pointwiseStatsOutput">
+			<attribute name="filename_template">pointwiseStats.nc</attribute>
+			<attribute name="type">output</attribute>
+			<attribute name="mode">forward;analysis</attribute>
+			<attribute name="output_interval">00:30:00</attribute>
+                        <attribute name="packages">pointwiseStatsAMPKG</attribute>
+                        <attribute name="clobber_mode">truncate</attribute>
+			<attribute name="runtime_format">single_file</attribute>
```

Comment:
If we don't choose to use the templates, we should add:
`<attribute name="io_type">netcdf</attribute>`
here to avoid performance issues associated with writing small files in parallel (#372).

Reply:
We should use them.

Reply:
Should be covered now...